.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Describe a bug with single-spa-angular
+---
+
+### Demonstration
+
+<!--
+Please provide the *STEPS TO REPRODUCE* and if possible a *MINIMAL DEMO* of the problem via
+https://stackblitz.com
+or
+https://codesandbox.io
+
+* Step 1
+* Step 2
+* Step 3
+
+-->
+
+### Expected Behavior
+
+### Actual Behavior
+
+### Environment
+
+<pre><code>
+Libs:
+- @angular/core version: X.Y.Z
```

Comment:
This information is unnecessary if they provide a code sandbox or stackblitz. Same goes for single-spa-angular version